src/main/java/software/aws/neptune/sparql/resultset/SparqlResultSetGetTypeInfo.java

```diff
@@ -0,0 +1,283 @@
+package software.aws.neptune.sparql.resultset;
+
+import software.aws.neptune.common.gremlindatamodel.resultset.ResultSetGetTypeInfo;
+
+import java.sql.Statement;
+import java.sql.Types;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SparqlResultSetGetTypeInfo extends ResultSetGetTypeInfo {
+    private static final Map<String, Object> XSDDECIMAL_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDINTEGER_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDNONPOSITIVEINTEGER_INFO = new HashMap<>();;
+    private static final Map<String, Object> XSDNONNEGATIVEINTEGER_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDPOSITIVEINTEGER_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDNEGATIVEINTEGER_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDBYTE_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDUNSIGNEDBYTE_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDDOUBLE_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDFLOAT_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDLONG_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDUNSIGNEDSHORT_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDUNSIGNEDINT_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDUNSIGNEDLONG_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDINT_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDSHORT_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDBOOLEAN_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDDATE_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDTIME_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDDATETIME_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDDATETIMESTAMP_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDDURATION_INFO = new HashMap<>();
+    private static final Map<String, Object> XSDSTRING_INFO = new HashMap<>();
+
+    private static final List<Map<String, Object>> TYPE_INFO = new ArrayList<>();
+
+    static {
```

Comment:
nit: There is a lot of repetition here, you could make a function in ResultSetGetTypeInfo that takes in the arguments and cut the code here down quite a bit.